config/config.go

```diff
@@ -401,6 +401,13 @@ func (r BlockRange) Validate() error {
 	return nil
 }
 
+type ConsensusAnalyzerConfig struct {
+	BlockBasedAnalyzerConfig `koanf:",squash"`
+
+	// AccountListRefresh is the config for the account list refresh analyzer.
+	AccountListRefresh ItemBasedAnalyzerConfig `koanf:"account_list_refresh"`
```

Comment:
I have "stuffed" this into the Consensus analyzer config instead of making a standalone top-level config for the account list analyzer. Also the analyzer doesn't need to be separately enabled, but is enabled automatically if consensus analyzer is enabled. This makes sense to me since this analyzer is tightly coupled with the consensus analyzer.

Let me know if you agree @Andrew7234